### DISCUSSION TITLE
Rog Ally bind trigger customer setting not working.

### DISCUSSION THREAD

**Discussion by tendowang823** [seerge/g-helper#3539] [Q&A]
AllyControl.cs
In 'static private void BindZone(BindingZone zone)' funcion

                default:
                    KeyL1 = AppConfig.GetString("bind_trl", desktop ? BindShiftTab : BindLT);
                    KeyR1 = AppConfig.GetString("bind_trr", desktop ? BindMouseR : BindRT);
                    KeyL2 = AppConfig.GetString("bind2_trl");
                    KeyR2 = AppConfig.GetString("bind2_trr");
                    break;

These parameter names 'bind_trl' 'bind_tr' 'bind2_trl' 'bind2_trr' seem incorrect.

in config.json data
The saved parameter names are 'bind_lt' 'bind_rt'  'bind2_lt' 'bind2_rt'

This seems to be the problem.

Please forgive my bad English.

**Comment by seerge** [comment]
@tendowang823 hello, thanks for spotting. This build should read correct bindings for triggers.

[GHelper.zip](https://github.com/user-attachments/files/18309879/GHelper.zip)
